addons/spreadsheet/static/src/scss/spreadsheet_dashboard.scss

```
@@ -0,0 +1,3 @@
+.o-sample-dashboard-opacity-40 .o-figure {
```

```suggestion
.o-sample-dashboard .o-figure {
```
Nitpick. We might want to add more CSS to sample dashboard later, in that case the `opacity-40` won't make sense.

Also wasnt' there some talk of not graying some types of charts (maybe I remember wrong) ?

---

As far as I know, there are no exceptions

---

Done ✅